projects/schematics/src/shared/lib-configs/integration-libs/cdc-schematics-config.ts

```diff
@@ -119,6 +120,54 @@ export const CDC_SCHEMATICS_CONFIG: SchematicConfig = {
     },
   ],
 };
+export const CDC_SCHEMATICS_CONFIG: SchematicConfig = {
+  library: {
+    featureName: CDC_FEATURE_NAME,
+    mainScope: SPARTACUS_CDC,
+  },
+  folderName: CDC_FOLDER_NAME,
+  moduleName: CDC_MODULE_NAME,
+  featureModule: [
+    {
+      importPath: SPARTACUS_CDC,
+      name: CDC_MODULE,
+    },
+    {
+      name: CDC_USER_ACCOUNT_MODULE,
+      importPath: SPARTACUS_CDC_USER_ACCOUNT,
+    },
+    {
+      name: CDC_USER_PROFILE_MODULE,
+      importPath: SPARTACUS_CDC_USER_PROFILE,
+    },
+  ],
+  lazyLoadingChunk: {
+    moduleSpecifier: SPARTACUS_CDC_ROOT,
+    namedImports: [CDC_FEATURE_CONSTANT],
+  },
+  rootModule: {
+    importPath: SPARTACUS_CDC_ROOT,
+    name: CDC_ROOT_MODULE,
+    content: `${CDC_ROOT_MODULE}`,
+  },
+  customConfig: buildCdcConfig,
+  i18n: {
+    resources: CDC_TRANSLATIONS,
+    chunks: CDC_TRANSLATION_CHUNKS_CONFIG,
+    importPath: SPARTACUS_CDC_ASSETS,
+  },
+  dependencyFeatures: [USER_PROFILE_FEATURE_NAME],
+  importAfter: [
+    {
+      markerModuleName: USER_ACCOUNT_MODULE,
+      featureModuleName: CDC_USER_ACCOUNT_MODULE,
+    },
+    {
+      markerModuleName: USER_PROFILE_MODULE,
+      featureModuleName: CDC_USER_PROFILE_MODULE,
+    },
+  ],
+};
```

Comment:
[QUESTION]
Is that true that everything that is related to B2C is also included in B2B? In other words, I is the CDC B2B a superset of features in CDC B2C?

I yes, then copying the whole config to a separate variable (while a lot of config parts are shared) is prone to human errors in the future - we might forget to add new CDC features to both CDC and CDC-B2B Configs in the future, but add it just in B2C or just in B2B.

What do you think about defining a shared cofing both for B2B and B2C and reuse it in both configs? 
OR just reusing parts of the B2C config in the B2B config? Thanks to that, any addition to B2C will be also reflected in B2B. And if you want to add something only for B2B, then we can add it just to B2B config.
What do you think?